e2e/specs/age-based-vitals.spec.ts

```diff
@@ -0,0 +1,597 @@
+
+    await test.step('When I fill `37.8` as the temperature', async () => {
+      await vitalsPage.page.getByRole('spinbutton', { name: /temperature/i }).fill('37.8');
+    });
+
+    await test.step('And I fill `97` as the systolic', async () => {
+      await vitalsPage.page.getByRole('spinbutton', { name: /systolic/i }).fill('97');
+    });
+
+    await test.step('And I fill `57` as the diastolic', async () => {
+      await vitalsPage.page.getByRole('spinbutton', { name: /diastolic/i }).fill('57');
+    });
+
+    await test.step('And I fill `11` as the respiration rate', async () => {
+      await vitalsPage.page.getByRole('spinbutton', { name: /respiration rate/i }).fill('11');
+    });
+
+    await test.step('And I click on the `Save and close` button', async () => {
+      await vitalsPage.page.getByRole('button', { name: /save and close/i }).click();
+    });
+
+    await test.step('Then I should see a success notification', async () => {
+      await expect(vitalsPage.page.getByText(/vitals and biometrics saved/i)).toBeVisible();
+    });
+
+    await test.step('And I should see the newly recorded vital signs on the page', async () => {
+      await expect(headerRow).toContainText(/temp/i);
+      await expect(headerRow).toContainText(/bp/i);
+      await expect(headerRow).toContainText(/r. rate/i);
+      await expect(dataRow).toContainText('37.8');
+      await expect(dataRow).toContainText('97 / 57');
+      await expect(dataRow).toContainText('11');
+      const highRange = vitalsPage.page.getByRole('cell', { name: '37.8' });
+      const highRangeRate = vitalsPage.page.getByRole('cell', { name: '11' });
+
+      const backgroundColor = await highRange.evaluate((el) => window.getComputedStyle(el).backgroundColor);
+      expect(backgroundColor).toBe('rgb(255, 242, 232)');
+
+      const afterContent = await highRange.evaluate((el) => {
+        const after = window.getComputedStyle(el, '::after');
+        return after.content;
+      });
+
+      expect(afterContent).toBe('" ↑"');
+
+      const backgroundColorLow = await highRangeRate.evaluate((el) => window.getComputedStyle(el).backgroundColor);
+      expect(backgroundColorLow).toBe('rgb(255, 242, 232)');
+
+      const afterContentLow = await highRangeRate.evaluate((el) => {
+        const after = window.getComputedStyle(el, '::after');
+        return after.content;
+      });
+      expect(afterContentLow).toBe('" ↓"');
+    });
+  });
+});
+
+test.afterEach(async ({ api }) => {
+  if (newbornVisit) {
+    try {
+      await endVisit(api, newbornVisit);
+    } catch (e) {
+      console.warn('Failed to delete newborn visit:', e);
+    }
+  }
+
+  if (infantVisit) {
+    try {
+      await endVisit(api, infantVisit);
+    } catch (e) {
+      console.warn('Failed to delete infantVisit:', e);
+    }
+  }
+
+  if (childVisit) {
+    try {
+      await endVisit(api, childVisit);
+    } catch (e) {
+      console.warn('Failed to delete childborn visit:', e);
+    }
+  }
```

Comment:
Nit: missing a new line here
```suggestion
  }
  
```